hypervisor/include/public/acrn_hv_defs.h

```diff
@@ -281,6 +282,32 @@ struct hc_api_version {
 	uint32_t minor_version;
 } __aligned(8);
 
+/**
+ * Hypervisor api, return it for HC_GET_PLATFORM_INFO hypercall
```

Comment:
```suggestion
 * Hypervisor API, return it for HC_GET_PLATFORM_INFO hypercall
```

I feel it'd be clearer if we used `API` (capital letters) instead of `api` in this file.